api/test_api.py

```diff
@@ -1,2 +1,27 @@
-# pylint: ignore-pylint
-from api import *
+# pylint: skip-file
+from dotenv import load_dotenv
+import pytest
+from api import app
+from unittest.mock import patch
+
+load_dotenv()
+
+
+@pytest.fixture
+def client():
+    app.config["TESTING"] = True
+    with app.test_client() as client:
+        yield client
```

Comment:
Add this to a conftest file

Reply:
As of now this is the only fixture, so a separate file does not seem the most appropriate. I am willing to change this at a later time if the need arises.